src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+
+  const totalCost = days * dailyRate;
+
+  let discount = 0;
+
+  if (days >= 3) {
+    discount = 20;
+  }
+
+  if (days >= 7) {
+    discount = 50;
+  }
```

Comment:
Logical issue: The current discount logic overrides the $20 discount with the $50 discount for rentals of 7 days or more. Consider using an `else if` statement for the 7-day condition to ensure that only the appropriate discount is applied.